api/v1alpha1/condition_consts.go

```diff
@@ -105,4 +111,12 @@ const (
 	// VirtualMachineImageV1Alpha1NotCompatibleReason (Severity=Error) documents that the VirtualMachine Image
 	// is not prepared for VMService consumption.
 	VirtualMachineImageV1Alpha1NotCompatibleReason = "VirtualMachineImageV1Alpha1NotCompatible"
+
+	// VirtualMachineImageProviderNotReadyReason (Severity=Error) documents that the VirtualMachine Image provider
+	// from the vSphere content library is not ready.
```

Comment:
Is the provider here the `vSphere content library` or some provider from vSphere content library? just to understand what this provider means here.